html/management.html

```diff
@@ -1496,24 +1591,9 @@ <h5 class="modal-title" data-i18n="wifi.delete.title"></h5>
 		}	
 	}
 
-	async function updateWifiFields() {
-		let hostnameElem = document.getElementById("hostname");
-		let scanWifiElem = document.getElementById("scan_wifi_on_start");
-
-		let config = await (await fetch("/wificonfig")).json();
-
-		if (config.hostname && config.hostname !== "") {
-			hostnameElem.value = config.hostname;
-			hostnameElem.defaultValue = config.hostname;
-		} 
-		scanWifiElem.checked = config.scanwifionstart;
-	}
-
 	function onWifiTabOpened() {
-		console.log("wifi tab opened!")
+		console.log("wifi tab opened!");
 		rebuildSSIDList();
```

Comment:
I have leave it as is at the moment because the call rebuildSSIDList() makes a new HTTP request to ESP-32. This can be time critical loading the UI with it's settings. Now it's loaded only if WiFi settings be opened

Reply:
I meant to remove that HTTP call and include the list of SSIDs with the rest of the settings sent over Websocket.

Reply:
@SZenglein Ah OK, i will take a closer look at this point again!